app/views/welcome/index.html.erb

```diff
@@ -1,3 +1,7 @@
+<h2>
+	<%= link_to "Complete Full Profile->", profile_path %>
```

Comment:
This link placement needs to be thought through, it currently looks a little out of place.